src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorParameter.cs

```diff
@@ -22,30 +22,31 @@ public class ConnectorParameter : ConnectorSchema
 
         internal bool IsBodyParameter = false;
 
-        internal ConnectorParameter(OpenApiParameter openApiParameter, ConnectorCompatibility compatibility)
-            : this(openApiParameter, null, false, compatibility)
+        internal ConnectorParameter(OpenApiParameter openApiParameter, SymbolTable optionSets, ConnectorCompatibility compatibility)
```

Comment:
>SymbolTable [](http://example.com/codeflow?start=71&length=11)

Can this be ReadOnlySymbolTable?
ie, are we mutating this?

Reply:
I've removed it